ui/raidboss/data/05-shb/trial/varis-ex.txt

```diff
@@ -0,0 +1,116 @@
+# Memoria Misera Extreme
+# -ic "Phantom Varis" -ii 4CEF 4CF1 4CDD 4CEE
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# Phase 1
+0 "Start!"
+0.0 "--sync--" sync /:Engage!/ window 0,1
+0.9 "--sync--" sync /:Varis Yae Galvus:4CEF:/ window 1,0
+19.0 "Altius" sync /:Varis Yae Galvus:4CCA:/
+28.9 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+34.1 "Terminus Est" # sync /:Terminus Est:4CB4:/
+45.1 "Citius" sync /:Varis Yae Galvus:4CF0:/
+56.0 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+64.2 "Ignis Est Action"
+64.2 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+82.4 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/
+91.5 "Altius" sync /:Varis Yae Galvus:4CCA:/
+100.4 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+101.6 "Terminus Est" # sync /:Terminus Est:4CB4:/
+108.5 "Ignis Est Action" # sync /:Ignis Est:4CB6:/
+109.6 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+120.7 "Reinforced Gunshield" sync /:Varis Yae Galvus:4CD9:/
+128.9 "Citius" sync /:Varis Yae Galvus:4CF0:/
+132.0 "Magitek Shielding" sync /:Varis Yae Galvus:4CDB:/
+147.2 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/
+156.0 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+163.6 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+164.2 "Ventus Est Action"
+166.7 "Magitek Burst Hit"
+177.1 "Vivere Militare Est" sync /:Varis Yae Galvus:4CCC:/
+
+# Phase 2
+200.1 "--targetable--"
+233.2 "Vivere Militare Est" sync /:Bladesblood:4CCF:/ duration 30 window 80,50
+
+# Phase 3 (random groups)
+262.3 "Gunshield Actions" # fake action here for placeholding
+
+
+# ================= random groups ===================
+# group 1 (Loaded Gunshield, spread)
+300.0 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/ window 100,400
+308.9 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+313.2 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+314.7 "Terminus Est"
+316.3 "Magitek Burst" sync /:Varis Yae Galvus:4CDF:/
+317.0 "Ignis Est Action"
+324.3 "Terminus Est" # sync /:Terminus Est:4CB4:/
+324.5 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+# since trigger would notice tankbuster with startsUsing, timeline would not show it
+# 312.7 "Citius" sync /:Varis Yae Galvus:4CF0:/
```

Comment:
Can you expand on this? I am a little confused by why this is not used. (Does it actually happen right before Terminus Est, or is the 312.7 timestamp wrong?)